### PR TITLE
[jit][await] Apply review comments

### PR DESCRIPTION
**Pull request by IvanKobzarev**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #93284

Differential Revision: [D42849920](https://our.internmc.facebook.com/intern/diff/D42849920)

### CODE REVIEW

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D42849920).

**Review by malfet — APPROVED** [review]

File: docs/source/index.rst [hunk 1]
```diff
@@ -81,7 +81,6 @@ Features described in this documentation are classified by release status:
    torch.autograd <autograd>
    torch.library <library>
    cuda
-   torch._awaits <_awaits>
```
Comment (malfet) [inline docs/source/index.rst:84]: We can probably leave this one in place as @svekars is working on PR to mark everything that starts with `_` as beta

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D42849920).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D42849920).

**Comment by IvanKobzarev** [comment]
@ivankobzarev has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D42849920).

**Pull request closed without merging**